src/printing.cpp

```
@@ -719,4 +719,369 @@ std::ostream& operator<<(std::ostream& os, const btf_line_info_t& line_info) {
     os << "; " << line_info.source_line << "\n";
     return os;
 }
+
+void print_invariants_filtered(std::ostream& os, const Program& prog, const bool simplify, const AnalysisResult& result,
```

Clang (14.0.6)

[warning] parameter name 'os' is too short, expected at least 3 characters

(readability-identifier-length)